tox.ini

```diff
@@ -21,9 +21,8 @@ deps =
     test: pytest-cov
     test: pytest-timeout
 commands_pre =
-    # Build ioutil for both current python (e.g. 3.8) and platform python (e.g. 3.6)
+    # Build ioutil for current python (e.g. 3.8)
```

Comment:
There was a reason for building ioutil for both current and platform python. Please find what was the reason in git history before removing this, and explain why it is ok to do this now.

Reply:
I'll do.

For the record, this is the commit: https://github.com/oVirt/ovirt-imageio/commit/b76751afaf3e260b638a501458be3d61912dfa6e